content/docs/alerting/alertmanager.md

```diff
@@ -59,8 +59,9 @@ alerts are checked whether they match all the equality or regular expression
 matchers of an active silence.
 If they do, no notifications will be sent out for that alert.
 
-Silences are configured in the web interface of the Alertmanager.
-
+Silences are configured in the web interface of the Alertmanager.  Note that because 
+silence state is updated on each group interval, it may take a few minutes for 
+newly-created silences to be reflected in the UI.
```

Comment:
Very minor.

`Silence state is updated on each group interval, so it may take a few minutes for newly-created silences to be reflected in the UI.` 

I always try to avoid saying "note that" if I can avoid in favor of succinct and clear (http://blog.apastyle.org/apastyle/2015/09/principles-of-writing-how-to-avoid-wordiness.html).